spec/rails_helper.rb

```diff
@@ -7,6 +7,11 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+require 'rake'
+Rails.application.load_tasks
+Rake::Task["db:migrate"].invoke
+Rake::Task["db:seed"].invoke
+
```

Comment:
I like this - it's definitely an improvement. What about adding a statement to explicitly remove all contents from the database just to be 110% sure we start off with the expected test data?

Reply:
interesting idea.

Reply:
Did some research;  I think we'd have to code "unseed" methods and/or drop and re-create the whole test db.  That seems a bit heavy handed.  If you think this is worth the trouble, I suggest you create a separate ticket for it;  I'm not feeling passionate about it.

Reply:
👍  Created #169